src/graphql.rs

```diff
@@ -19,13 +20,23 @@ use async_graphql::{
 };
 use base64::{engine::general_purpose::STANDARD as base64_engine, Engine};
 use chrono::{DateTime, TimeZone, Utc};
+use giganto_client::ingest::Packet as pk;
+use libc::timeval;
+use pcap::{Capture, Linktype, Packet, PacketHeader};
 use serde::{de::DeserializeOwned, Serialize};
-use std::{net::IpAddr, path::PathBuf};
-
-use self::network::NetworkFilter;
+#[cfg(target_os = "macos")]
+use std::os::fd::AsRawFd;
+#[cfg(target_os = "linux")]
+use std::os::unix::io::AsRawFd;
+use std::{
+    io::{Read, Seek, SeekFrom, Write},
+    net::IpAddr,
+    path::PathBuf,
+    process::{Command, Stdio},
+};
+use tempfile::tempfile;
 
 pub const TIMESTAMP_SIZE: usize = 8;
```

Comment:
Before and after a struct definition, insert an empty line, please

Reply:
Done